index.html

```diff
@@ -45,6 +46,193 @@ <h1 class="heading__title fw-700">I'm Adeyemi.<br /> Glad to see you!</h1>
+                me.
+            </p>
+            <p class="heading__connect">LET’S CONNECT</p>
+            <div class="heading__icons">
+                <ul class="heading__icons__ul ul flex">
+                    <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                                src="assets/images/twitter.svg" alt="twitter icon"></a></li>
+                    <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                                src="assets/images/linkedin.svg" alt="linkedin icon"></a></li>
+                    <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                                src="assets/images/medium.svg" alt="github icon"></a></li>
+                    <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                                src="assets/images/github.svg" alt="github icon"></a></li>
+                    <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                                src="assets/images/angellist.svg" alt="angellist icon"></a></li>
+                </ul>
+            </div>
+        </div>
+
+        <ul class="casualose ul">
+            <li>
+                <div class="header">
+                    <span>Language</span>
+                    <img src="assets/images/arrow_down.svg" alt="Down Arrow">
+                </div>
+    
+                <img class="is_desktop" src="assets/images/languages_desktop.svg" alt="Down Arrow">
+                <img class="is_mobile full_width" src="assets/images/languages_mobile.svg" alt="Down Arrow">
+    
+            </li>
+            <li>
+                <div class="header border-bottom">
+                    <span>Frameworks</span>
+                    <img src="assets/images/arrow_right.svg" alt="Right arrow">
+                </div>
+            </li>
+            <li>
+                <div class="header border-bottom">
+                    <span>Skills</span>
+                    <img src="assets/images/arrow_right.svg" alt="Right arrow">
+                </div>
+            </li>
+       </ul>
+        
+    </section>
+
+    <section class="footer flex">
+
+        <h1 class="title">Contact me</h1>
+
+        <p class="desc">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!</p>
+        
+        <!-- <form class="contact-form"> -->
+            <input placeholder="Lorem Ipsum" name="name" type="text" />
+            <input placeholder="Lorem Ipsum" name="name" type="text" />
+            <textarea rows="6" placeholder="Write your message here..." name="textss" ></textarea>
+            <div class="contact-button">
+                <button class="button-outline">Get in touch</button>
+            </div>
+            
+        <!-- </form> -->
+    </section>
```

Comment:
- [ ] Kindly remove this part because it a part of the next milestone , you can save it in a new branch for next milestone 👍 
![Capture 00](https://user-images.githubusercontent.com/80458199/235118993-642eab8e-1d7a-494d-8e06-5ec328e48e90.PNG)